apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -188,25 +214,30 @@ public synchronized void refreshFilters() {
      */
     try {
       // If new batches are present, then build new filters for the affected files
-      final Date currentLastDatabaseUpdate = fetchLastLoadedBatchTime();
-      if (this.transactionTime == null || this.transactionTime.before(currentLastDatabaseUpdate)) {
+      final Date currentLastBatchCreated =
+          fetchLastLoadedBatchCreated().orElse(BEFORE_LAST_UPDATED_FEATURE);
+      if (this.lastBatchCreated == null || this.lastBatchCreated.before(currentLastBatchCreated)) {
         LOGGER.info(
             "Refreshing LoadedFile filters with new filters from {} to {}",
-            transactionTime,
-            currentLastDatabaseUpdate);
-        List<LoadedTuple> loadedTuples = fetchLoadedTuples(this.transactionTime);
+            lastBatchCreated,
+            currentLastBatchCreated);
+        List<LoadedTuple> loadedTuples = fetchLoadedTuples(this.lastBatchCreated);
         this.filters = updateFilters(this.filters, loadedTuples, this::fetchLoadedBatches);
-        this.transactionTime = currentLastDatabaseUpdate;
+        this.lastBatchCreated = currentLastBatchCreated;
 
         // If batches been trimmed, then remove filters which are no longer present
-        final Date currentFirstBatchUpdate = fetchFirstLoadedBatchTime();
-        if (this.firstBatchUpdate == null
-            || this.firstBatchUpdate.before(currentFirstBatchUpdate)) {
+        final Date currentFirstBatchUpdate =
+            fetchFirstLoadedBatchCreated().orElse(BEFORE_LAST_UPDATED_FEATURE);
+        if (this.firstBatchCreated == null
+            || this.firstBatchCreated.before(currentFirstBatchUpdate)) {
           LOGGER.info("Trimmed LoadedFile filters before {}", currentFirstBatchUpdate);
```

Comment:
Same as above, this is currently logging every second in test.  Can this be changed to debug?